ch02/README.md

```diff
@@ -10,3 +10,5 @@
 - [02_bonus_bytepair-encoder](02_bonus_bytepair-encoder) contains optional code to benchmark different byte pair encoder implementations
 
 - [03_bonus_embedding-vs-matmul](03_bonus_embedding-vs-matmul) contains optional (bonus) code to explain that embedding layers and fully connected layers applied to one-hot encoded vectors are equivalent.
+
+- [04_bonus_dataloader-intuition](04_bonus_dataloader-intuition) contains optional (bonus) code to implement data loader with sliding window intuition.
```

Comment:
```suggestion
- [04_bonus_dataloader-intuition](04_bonus_dataloader-intuition) contains optional (bonus) code to explain the data loader more intuitively with simple numbers rather than text.
```